owncloud/owncloud.py

```diff
@@ -950,6 +950,57 @@ def user_is_in_group (self, user_name, group_name):
             return False
 
 
+    def get_user(self, user_name):
+        """Retrieves information about a user
+
+        :param user_name:  name of user to query
+        
+        :returns: Dictionary of information about user
+        :raises: ResponseError in case an HTTP error status was returned
+        """
+        res = self.__make_ocs_request(
+            'GET',
+            self.OCS_SERVICE_CLOUD,
+            'users/' + user_name,
```

Comment:
So far special characters were never allowed in user names in core, but I prefer to be on the safe side in case it changes one day. Please url encode the user_name with `urllib.quote()`.